### PR TITLE
Fix Vale typos

### PR DESCRIPTION
**Pull request by rrrutledge**

### CODE REVIEW

File: workbook/01-introduction.asciidoc [hunk 1]
```diff
@@ -105,7 +105,7 @@ More than one answer may be correct in some questions.
 
 ====== Correct answer: 1
 
-Why 1 is correct: The host teams open their code base to others and put effort into vetting contributions precisely because their code end up better and more featureful than if they did all the coding themselves.
```
Comment (rrrutledge) [inline workbook/01-introduction.asciidoc:108]: This is my favorite correction ... `featureful` 😜.

**Comment by rrrutledge** [comment]
@spier there is just one issue left that Vale has flagged, and I'm not sure how to correct it.  Can you take a look?  It is some markup that it interprets as an incorrect spelling of `InnerSource`.

File: workbook/01-introduction.asciidoc (hunk 1 above)
Comment (spier) [reply workbook/01-introduction.asciidoc:108]: Hehe yeah, I saw this one too.
I think `featureful` should be a real word :)

**Comment by spier** [comment]
> @spier there is just one issue left that Vale has flagged, and I'm not sure how to correct it. Can you take a look? It is some markup that it interprets as an incorrect spelling of `InnerSource`.

Very cool that you did all these fixes. Great job @rrrutledge!

I also saw the issue that you mentioned in [contributor/02-becoming-a-contributor-article.asciidoc](https://github.com/InnerSourceCommons/InnerSourceLearningPath/blob/main/contributor/02-becoming-a-contributor-article.asciidoc?plain=1#L31).

vale cannot understand the in-word italic formatting of Inner _Source_ in asciidoc.
_Aside_: Looks like such formatting cannot even be expressed here in GitHub markdown in one word at all.

An option could be to switch off the specific vale rule just for the line in that file.
[Looks like](https://vale.sh/docs/topics/config/#asciidoc) it should be possible. I will test this in your branch here shortly, hope you don't mind :)

**Comment by spier** [comment]
See the last commit for the fix I applied.

It looks like it worked 🥳 

The **Filtered Findings** section of the vale report now show [0 issues](https://github.com/InnerSourceCommons/InnerSourceLearningPath/actions/runs/6005095210/job/16287119751), as compared to [1 issue](https://github.com/InnerSourceCommons/InnerSourceLearningPath/actions/runs/6004778024/job/16286119875) that was there before.

I cannot confirm if the changes I made create any regressions in the rendered version of the LearningPath on the website. @rrrutledge How to check that?

**Comment by rrrutledge** [comment]
Thanks, @spier ❗️ I’ll try it out

**Comment by rrrutledge** [comment]
It works!  Translates to an HTML comment just fine.  Will merge now.

**Pull request merged**